blues/moves.py

```diff
@@ -177,70 +178,304 @@ def move(self, context):
                  single_move.move(context)
 
 
-class MoveEngine(object):
-    """MoveEngine provides perturbation functions for the context during the NCMC
-    simulation.
-    Ex.
-        from blues.ncmc import MoveEngine
-        probabilities = [0.25, 0.75]
-        #Where move is a list of two Move objects
-        mover = MoveEngine(move, probabilities)
-        #Get the dictionary of proposed moves
-        mover.moves
-    """
-    def __init__(self, moves, probabilities=None):
-        """Initialize the MovePropsal object that contains functions to perturb
-        the context in the NCMC simulation.
-        Parameters
-        ----------
-        moves : blues.ncmc.Model object or list of n blues.ncmc.Model-like objects
-            Specifies the possible moves to be performed.
+class SmartDartMove(RandomLigandRotationMove):
+    def __init__(self, structure, basis_particles, dart_size=0.2*unit.nanometers, resname='LIG'):
+        super(SmartDartMove, self).__init__(structure, resname=resname)
+        self.dartboard = []
+        self.n_dartboard = []
+        self.particle_pairs = []
+        self.particle_weights = []
+        self.basis_particles = basis_particles
+        self.dart_size = dart_size
+        self.calculateProperties()
+
+    def dartsFromParmEd(self, coord_files, topology=None):
+        """
+        Used to setup darts from a generic coordinate file, through MDtraj using the basis_particles to define
+        new basis vectors, which allows dart centers to remain consistant through a simulation.
+        This adds to the self.n_dartboard, which defines the centers used for smart darting.
```

Comment:
I have some problems reading this doc string, in part because you haven't introduced your terminology anywhere yet, @sgill2 . Specifically, what's a "dart" (OK, I know that part, but a naive user might not) a "dart center", a dartboard, etc.? 

You'll have to decide one place where you want to explain this, and then refer to it elsewhere. For example, maybe you want a `MOVES.md` file which explains the different types of moves implemented, how they work, and how to use them; you can link to this from the main `README.md`. For the other types of moves so far maybe this can be very brief, but it seems like for smart darting moves you'd need a little more. Then your doc strings here could just link to that.